rskj-core/src/main/java/co/rsk/RskContext.java

```diff
@@ -2142,7 +2261,13 @@ private MiningConfig getMiningConfig() {
                     rskSystemProperties.coinbaseAddress(),
                     rskSystemProperties.minerMinFeesNotifyInDollars(),
                     rskSystemProperties.minerGasUnitInDollars(),
-                    rskSystemProperties.minerMinGasPrice(),
+                    new MinGasPriceProvider(
+                            rskSystemProperties.minerStableGasPriceEnabled(),
+                            rskSystemProperties.minerMinGasPrice(),
+                            rskSystemProperties.minerStableGasPriceMinStableGasPrice(),
+                            rskSystemProperties.minerStableGasPriceRefreshRate(),
+                            rskSystemProperties.minerStableGasPriceSources()
+                    ),
```

Comment:
as discussed earler @Vovchyk , I'll move this to the MinGasPriceCalculator instantiation, passing the ethModules as well. Thought, the latter will be done in the [eth_call-related PR](https://github.com/rsksmart/rskj/pull/2372).